haskell-src/Concordium/Types/ProtocolLevelTokens/CBOR.hs

```diff
@@ -1572,6 +1587,8 @@ decodeTokenModuleState = decodeMap decodeVal build Map.empty
     convertTokenMetadataUrl :: CBOR.Term -> Maybe TokenMetadataUrl
     convertTokenMetadataUrl = either (const Nothing) Just . decodeTokenMetadataUrlHelper
 
+    convertAccountAddress = error "TODO(drsk) how are account addresses encoded?"
```

Comment:
@soerenbf Do you know how account addresses are encoded/decoded?

Reply:
We have `encodeAccountAddress :: AccountAddress -> Encoding` defined, which seems to match what you're looking for? For other types we represent accounts as `TokenHolder` which is encoded differently. I don't know if that makes sense here though.

@td-concordium what do you think?